addons/survey/static/src/js/survey_form.js

```diff
@@ -109,6 +109,15 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
     _onKeyDown: function (event) {
         var self = this;
 
+        if (self.options.questionsLayout === "one_page" && !self.options.isStartScreen) {
+            if (this.$("input").is(":focus") && event.key === "Enter") {
+                event.preventDefault();
+            }
```

Comment:
When you focus on an input tag and press "Enter", the page gets submitted, So for that, I have to add that condition

Reply:
1. I have to add that condition to prevent submission of the survey on "Enter" key.
2. We are allowed to press "Enter" on other layouts, so there is no issue there.